build_tools/circle/list_versions.py

```diff
@@ -32,10 +33,19 @@ def human_readable_data_quantity(quantity, multiple=1024):
             quantity /= multiple
 
 
-def get_pdf_size(version):
+def get_file_extension(version):
```

Comment:
We need handle the `version='dev'` case explicitly:

```py
def get_file_extension(version):
    if version == "dev":
        return "zip"

    current_version = LooseVersion(version)
    min_zip_version = LooseVersion("1.0.0")

    return 'zip' if current_version >= min_zip_version else 'pdf'
```

The `'.'` should be removed because caller already includes it.